packages/react-components/react-storybook-addon/src/preset/manager.ts

```diff
@@ -3,7 +3,7 @@ import { addons, types } from '@storybook/manager-api';
 import { ADDON_ID, DIR_ID, STRICT_MODE_ID, THEME_ID } from '../constants';
 import { ThemePicker } from '../components/ThemePicker';
 import { ReactStrictMode } from '../components/ReactStrictMode';
-import { DirSwitch } from '../components/DirSwitch';
```

Comment:
for whatever reason this is not shown as rename :D , so if you did not rename make sure to remove the DirSwitch file . ty

Reply:
👀